Makefile

```diff
@@ -60,14 +60,14 @@ MIXIN_OUT_PATH_SUFFIXES := "" "-baremetal"
 JSONNET_MANIFESTS_PATH := operations/mimir
 
 # Doc templates in use
-DOC_TEMPLATES := docs/sources/operators-guide/configure/reference-configuration-parameters/index.template
+DOC_TEMPLATES := docs/sources/mimir/operators-guide/configure/reference-configuration-parameters/index.template
 
 # Documents to run through embedding
-DOC_EMBED := docs/sources/operators-guide/configure/configure-the-query-frontend-work-with-prometheus.md \
-	docs/sources/operators-guide/configure/mirror-requests-to-a-second-cluster/index.md \
-	docs/sources/operators-guide/architecture/components/overrides-exporter.md \
-	docs/sources/operators-guide/get-started/_index.md \
-	docs/sources/operators-guide/deploy-grafana-mimir/jsonnet/deploy.md
+DOC_EMBED := docs/sources/mimir/operators-guide/configure/configure-the-query-frontend-work-with-prometheus.md \
+	docs/sources/mimir/operators-guide/configure/mirror-requests-to-a-second-cluster/index.md \
+	docs/sources/mimir/operators-guide/architecture/components/overrides-exporter.md \
+	docs/sources/mimir/operators-guide/get-started/_index.md \
+	docs/sources/mimir/operators-guide/deploy-grafana-mimir/jsonnet/deploy.md
```

Comment:
I will look into this later to fully understand what is going on, but thank you for finding these embeds. (I am wondering why these would need to be done with the `Makefile` and not via a Hugo shortcode. Out-of-scope for this PR.)